tests/unit/cellguide_pipeline/fixtures/cell_metadata.json

```diff
@@ -1 +1 @@
ogy, localization, and surface marker expression (CD123-positive) from other dendritic cell types and associated with early stage immune responses, particularly the release of physiologically abundant amounts of type I interferons in response to infection.", "synonyms": ["DC2", "IPC", "T-associated plasma cell", "interferon-producing cell", "lymphoid dendritic cell", "pDC", "plasmacytoid T cell", "plasmacytoid monocyte", "type 2 DC"]}, "CL:0000785": {"name": "mature B cell", "id": "CL:0000785", "clDescription": "A B cell that is mature, having left the bone marrow. Initially, these cells are IgM-positive and IgD-positive, and they can be activated by antigen.", "synonyms": ["mature B lymphocyte", "mature B-cell", "mature B-lymphocyte"]}, "CL:0001201": {"name": "B cell, CD19-positive", "id": "CL:0001201", "clDescription": "A B cell that is CD19-positive.", "synonyms": ["B lymphocyte, CD19-positive", "B-cell, CD19-positive", "B-lymphocyte, CD19-positive", "CD19+ B cell", "CD19-positive B cell"]}, "CL:0000946": {"name": "antibody secreting cell", "id": "CL:0000946", "clDescription": "A lymphocyte of B lineage that is devoted to secreting large amounts of immunoglobulin.", "synonyms": []}, "CL:0000789": {"name": "alpha-beta T cell", "id": "CL:0000789", "clDescription": "A T cell that expresses an alpha-beta T cell receptor complex.", "synonyms": ["alpha-beta T lymphocyte", "alpha-beta T-cell", "alpha-beta T-lymphocyte"]}, "CL:0002419": {"name": "mature T cell", "id": "CL:0002419", "clDescription": "A T cell that expresses a T cell receptor complex and has completed T cell selection.", "synonyms": ["CD3e-positive T cell", "mature T-cell"]}, "CL:0000813": {"name": "memory T cell", "id": "CL:0000813", "clDescription": "A long-lived, antigen-experienced T cell that has acquired a memory phenotype including distinct surface markers and the ability to differentiate into an effector T cell upon antigen reexposure.", "synonyms": ["memory T lymphocyte", "memory T-cell", "memory T-lymphocyte"]}, "CL:0000898": {"name": "naive T cell", "id": "CL:0000898", "clDescription": "Mature T cell not yet exposed to antigen with the phenotype CCR7-positive, CD45RA-positive, and CD127-positive. This cell type is also described as being CD25-negative, CD62L-high and CD44-low.", "synonyms": ["naive T lymphocyte", "naive T-cell", "naive T-lymphocyte"]}, "CL:0001200": {"name": "lymphocyte of B lineage, CD19-positive", "id": "CL:0001200", "clDescription": "A lymphocyte of B lineage that is CD19-positive.", "synonyms": []}, "CL:1001599": {"name": "pancreas exocrine glandular cell", "id": "CL:1001599", "clDescription": "Glandular cell of exocrine pancreas epithelium. Example: pancreatic acinar cell, glandular cells in pancreatic canaliculi, glandular cells in pancreatic ducts.", "synonyms": []}, "CL:1001433": {"name": "epithelial cell of exocrine pancreas", "id": "CL:1001433", "clDescription": "An epithelial cell of the exocrine pancreas.", "synonyms": ["exocrine pancreas cell"]}}
```

Comment:
Noticed a diff here in synonyms. OWLs/OBOs have two synonym types--related Synonyms and exact Synonyms. Pronto does not distinguish between the two, while owlready2 does. For COG, it only looks at exact synonyms. However, I see that the previous version of this file also included related Synonyms. 

without further context, I think it makes sense to only count exact synonyms (for example: "CL:0000084" / T-Cell used to count both mature T Cell and immature T Cell as synonyms because they are "related synonyms", which seems misleading to me at a glance. It does not include these in the COG metadata file). But I can also patch COG to achieve parity by counting both if you think it does make sense to consolidate or don't want to make that call as part of this work.